### ISSUE TITLE
Always crashes with '--python_version=3.5'

### ISSUE THREAD

**Issue by hasufell** [google/pytype#56]
```
$ pytype --python_version=3.5 foo.py
Traceback (most recent call last):
  File "/usr/host/bin/pytype", line 258, in <module>
    sys.exit(main(sys.argv) or 0)
  File "/usr/host/bin/pytype", line 201, in main
    return _run_pytype(options)
  File "/usr/host/bin/pytype", line 246, in _run_pytype
    print_errors=True)
  File "/usr/host/bin/pytype", line 131, in process_one_file
    options=options)
  File "/usr/host/bin/pytype", line 49, in check_pyi
    cache_unknowns=options.cache_unknowns)
  File "/usr/lib/python2.7/site-packages/pytype/infer.py", line 675, in check_types
    py_src, py_filename, init_maximum_depth, run_builtins)
  File "/usr/lib/python2.7/site-packages/pytype/vm.py", line 689, in run_program
    node, f_globals, f_locals = self.preload_builtins(node)
  File "/usr/lib/python2.7/site-packages/pytype/vm.py", line 652, in preload_builtins
    builtins_code = self.compile_src(src)
  File "/usr/lib/python2.7/site-packages/pytype/vm.py", line 637, in compile_src
    filename=filename, mode=mode)
  File "/usr/lib/python2.7/site-packages/pytype/pyc/pyc.py", line 158, in compile_src
    code = parse_pyc_string(pyc_data)
  File "/usr/lib/python2.7/site-packages/pytype/pyc/pyc.py", line 129, in parse_pyc_string
    return parse_pyc_stream(StringIO.StringIO(data))
  File "/usr/lib/python2.7/site-packages/pytype/pyc/pyc.py", line 109, in parse_pyc_stream
    python_version = magic.magic_word_to_version(magic_word)
  File "/usr/lib/python2.7/site-packages/pytype/pyc/magic.py", line 105, in magic_word_to_version
    return PYTHON_MAGIC[magic_word]
KeyError: '3351\nFile: foo.py'
```

Works fine with `--python_version=3.4` though.

The file examined is:

```py
def foo1(a: int) -> str:
    return (lambda x: x + 1)(a)

def foo2(a: int) -> str:
    return a + 1

def foo3(a: str) -> int:
    return (lambda x: x + 1)(a)
```

**Comment by hasufell** [comment]
Is this fixed?

**Comment by martindemello** [comment]
yes, fixed for 3.5 and 3.6
